Dockerfile.utils

```diff
@@ -0,0 +1,46 @@
+FROM python:3.8-slim
+
+ENV LANG=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Vienna
+
+ENV PYPI_USERNAME=<PYPI_USERNAME>
+ENV PYPI_PASSWORD=<PYPI_PASSWORD>
+
+RUN apt-get update -qq  \
+  && apt-get dist-upgrade -qq -y --no-install-recommends --fix-missing make tmux vim sed tree tk-dev curl sudo tzdata \
+  && ln -snf /usr/share/zoneinfo/$TZ /etc/localtime  \
+  && echo $TZ > /etc/timezone \
+  && dpkg-reconfigure --frontend noninteractive tzdata \
+  && apt-get clean \
+  && apt-get purge -y --auto-remove \
+  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN pip install --upgrade pip && pip install twine
+
+RUN curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+    && sudo apt update \
+    && sudo apt install -y gh
+
+WORKDIR /home/ondewo/
+
+COPY Makefile .
+
+COPY requirements.txt .
+
+COPY setup.py  .
+
+COPY setup.cfg .
+
+COPY README.md .
+
+COPY RELEASE.md .
+
+COPY ./envs/versions.env ./envs/versions.env
+
+COPY ondewo ondewo
+
+# run the container forever
+CMD tail -f /dev/null
```

Comment:
Needs newline at the end of file, does the repo have no `pre-commit-config.yaml`?